tests/url.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
```

Comment:
```suggestion

require 'English'
```

The "English" library must be required to use `$CHILD_STATUS` instead of `$?`

See https://github.com/rubocop/rubocop/issues/1747